setup.py

```diff
@@ -13,4 +21,6 @@
         "bitsandbytes",
     ],
     packages=find_packages(),
+    long_description=readme,
```

Comment:
negative
@lantiga

Reply:
don't bloat this codebase please...
let's keep all this engineering stuff out (@lantiga )

Reply:
ok, closing it; just be aware that the setup is not valid :)
also, let's state in the contribution section that we do not expect any code health contributions...